.github/workflows/e2e-tests.yml

```diff
@@ -80,218 +293,94 @@ jobs:
         shardTotal: [4]
         # TODO: add ref here for main and latest release tag
     runs-on: ${{ matrix.os }}
-    needs: conditions
     steps:
-    - uses: actions/create-github-app-token@v1
-      id: app-token
-      with:
-        app-id: ${{ secrets.MODELING_APP_GH_APP_ID }}
-        private-key: ${{ secrets.MODELING_APP_GH_APP_PRIVATE_KEY }}
-        owner: ${{ github.repository_owner }}
-    - uses: actions/checkout@v4
-      with:
-        token: ${{ steps.app-token.outputs.token }}
-    - id: filter
-      name: Check for Rust changes
-      uses: dorny/paths-filter@v3
-      with:
-        filters: |
-          rust:
-            - 'rust/**'
-    - uses: actions/setup-node@v4
-      if: needs.conditions.outputs.should-run == 'true'
-      with:
-        node-version-file: '.nvmrc'
-        cache: 'yarn'
-    - name: Install dependencies
-      id: deps-install
-      if: needs.conditions.outputs.should-run == 'true'
-      shell: bash
-      run: yarn
-    - name: Cache Playwright Browsers
-      if: needs.conditions.outputs.should-run == 'true'
-      uses: actions/cache@v4
-      with:
-        path: |
-          ~/.cache/ms-playwright/
-        key: ${{ runner.os }}-playwright-${{ hashFiles('yarn.lock') }}
-    - name: Install Playwright Browsers
-      if: needs.conditions.outputs.should-run == 'true'
-      shell: bash
-      run: yarn playwright install --with-deps
-    - name: Download Wasm Cache
-      id: download-wasm
-      if: ${{ needs.conditions.outputs.should-run == 'true' && github.event_name != 'schedule' && steps.filter.outputs.rust == 'false' }}
-      uses: dawidd6/action-download-artifact@v7
-      continue-on-error: true
-      with:
-        github_token: ${{secrets.GITHUB_TOKEN}}
-        name: wasm-bundle
-        workflow: build-and-store-wasm.yml
-        branch: main
-        path: rust/kcl-wasm-lib/pkg
-    - name: copy wasm blob
-      if: ${{ needs.conditions.outputs.should-run == 'true' && github.event_name != 'schedule' && steps.filter.outputs.rust == 'false' }}
-      shell: bash
-      run: cp rust/kcl-wasm-lib/pkg/kcl_wasm_lib_bg.wasm public
-      continue-on-error: true
-    - name: Build WASM condition
-      id: wasm
-      if: needs.conditions.outputs.should-run == 'true'
-      shell: bash
```

Comment:
I was wondering if this was deleted intentionally, and it seems like it _might_ be the same since the `set` line is there. https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

Reply:
Yes it was! Since this is now part of a workflow that `runs-on: namespace-profile-ubuntu-8-cores`